bundlesize.config.json

```diff
@@ -34,11 +34,11 @@
     },
     {
       "path": "static/build/page-edit.css",
-      "maxSize": "20KB"
+      "maxSize": "21KB"
```

Comment:
These are not related to this patchset, please rebase now that the master branch has been cleaned up and remove this change.

Reply:
Done.